MyListTest/UnitTest1.cs

```diff
@@ -0,0 +1,65 @@
+
+namespace MyListTest {
+    [TestClass]
+    public class UnitTest1 {
+        [TestMethod]
+        public void AddTest () {
+            MyList<int> mNumList = new ();
+            mNumList.Add (1);
+            mNumList.Add (2);
+            mNumList.Add (3);
+            mNumList.Add (4);
+            mNumList.Add (5);
+            Assert.AreEqual (5, mNumList.Count);
+            Assert.AreEqual (8, mNumList.Capacity);
+            Assert.AreEqual (1, mNumList[0]);
+            mNumList[2] = 6;
+            Assert.AreEqual (6, mNumList[2 ]);
+            Assert.ThrowsException<IndexOutOfRangeException> (() => mNumList[9] = 3, "Item not found in the list");
+
+        }
+        [TestMethod]
+        public void RemoveTest () {
+            MyList<int> numList = new ();
+            numList.Add (1);
+            numList.Add (2);
+            numList.Remove (1);
+            Assert.AreEqual (1, numList.Count);
+            Assert.ThrowsException<InvalidOperationException> (() => numList.Remove (5), "Item not found in the list");
+            Assert.ThrowsException<IndexOutOfRangeException> (() => numList[-2]);

```

Comment:
After modifying the Remove method implementation, the tests here also should be modified to check if this method returns correctly. There should not be any test for exception handling as this method will just return a boolean and doesn't throw any exception.